pydantic/errors.py

```diff
@@ -0,0 +1,270 @@
+from decimal import Decimal
+from typing import Union
+
+__all__ = (
```

Comment:
I think this will lead to problems when people (eg. me) add errors and forget to update `__all__`, let's remove `__all__`.

If we're really worried about exposing `Decimal` etc. we could even import them as `from decimal import Decimal as _Decimal`, but I don't think it's necessary.